src/appleseed/renderer/meta/tests/test_tracer.cpp

```diff
@@ -228,8 +228,8 @@ TEST_SUITE(Renderer_Kernel_Lighting_Tracer)
         Tracer                                  m_tracer;
 
         Fixture()
-          : m_trace_context(FixtureParams::FixtureBaseClass::m_scene)
-          , m_texture_store(FixtureParams::FixtureBaseClass::m_scene)
+          : m_trace_context(FixtureParams::FixtureBaseClass::m_project)
+          , m_texture_store(FixtureParams::FixtureBaseClass::m_project.get_texture_store())
```

Comment:
Here for example, the texture store was initially created without any parameters. With the changed you made to project, we end up having the same texture_store everywhere. Isn't it a problem ?

Reply:
What I could gather from my conversation with @dictoon on Discord is that,

Everywhere we used to create a `TextureStore` with default value of `ParamArray`, is for one-time use only and typically loads a single texture, so the params doesn't matter.

But still, I would like @dictoon to verify this.